Index: lib/config.json
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>{\r\n    \"xy_sensitivity\": 15.2,\r\n    \"targeting_sensitivity\": 50.0,\r\n    \"normal_scale\": 0.6578947368421053,\r\n    \"targeting_scale\": 1.3157894736842106,\r\n    \"mouse_scale\": 0.86,\r\n    \"auto_fire\": true,\r\n    \"slot_1_hotkey\": \"1\",\r\n    \"slot_2_hotkey\": \"2\",\r\n    \"slot_3_hotkey\": \"3\",\r\n    \"slot_4_hotkey\": \"4\",\r\n    \"slot_5_hotkey\": \"5\",\r\n    \"shotgun_slot\": 2\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/lib/config.json b/lib/config.json
--- a/lib/config.json	(revision 4495dba13e7e55ae26d9bb39db5d4c5658c193f3)
+++ b/lib/config.json	(date 1644123120749)
@@ -3,7 +3,7 @@
     "targeting_sensitivity": 50.0,
     "normal_scale": 0.6578947368421053,
     "targeting_scale": 1.3157894736842106,
-    "mouse_scale": 0.86,
+    "mouse_scale": 0.87,
     "auto_fire": true,
     "slot_1_hotkey": "1",
     "slot_2_hotkey": "2",
